db/migrate/20240724021732_add_login_method_to_user_auth_tokens.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLoginMethodToUserAuthTokens < ActiveRecord::Migration[7.1]
+  def change
+    add_column :user_auth_tokens, :login_method, :string, default: Auth::LOGIN_METHOD_LOCAL
```

Comment:
I think we should set a limit on the string column just to be safe.

Reply:
`Auth::LOGIN_METHOD_LOCAL` is application code and if someone decides to change it in the future, the default may differ between rows.